src/Paket.Core/PaketConfigFiles/DependenciesFile.fs

```diff
@@ -229,6 +229,7 @@ type DependenciesFile(fileName,groups:Map<GroupName,DependenciesGroup>, textRepr
                 RemoteDownload.DownloadSourceFiles(Path.GetDirectoryName fileName, groupName, force, remoteFiles)
 
             // Step 1 Package resolution
```

Comment:
let's call this something other than `step 1` & `step 2`, it's already confusing enough with all the other steps